### PR TITLE
Remove an unused local variable

### PR DESCRIPTION
**Pull request by songy23**

### CODE REVIEW

File: impl_core/src/test/java/io/opencensus/implcore/trace/export/NoopRunningSpanStoreImplTest.java [hunk 1]
```diff
@@ -66,17 +66,16 @@
...
-            spanContext,
-            recordSpanOptions,
-            SPAN_NAME,
-            null,
-            false,
-            TraceParams.DEFAULT,
-            startEndHandler,
-            timestampConverter,
-            testClock);
+    SpanImpl.startSpan(
```
Comment (sebright) [inline impl_core/src/test/java/io/opencensus/implcore/trace/export/NoopRunningSpanStoreImplTest.java]: I'm not sure I understand this change.  Does creating the span have a side effect?  I think it is more likely that the new span should be assigned to `spanImpl`, which is null.

I also wonder if RunningSpanStoreImpl.onStart and onEnd should have rejected the null `spanImpl`.

Comment (songy23) [reply impl_core/src/test/java/io/opencensus/implcore/trace/export/NoopRunningSpanStoreImplTest.java]: I don't have context on this test. The local variable `SpanImpl span` is never used and failed the build when import. 

@HailongWen should this be assigned to `private SpanImpl spanImpl;` instead?

Comment (bogdandrutu) [reply impl_core/src/test/java/io/opencensus/implcore/trace/export/NoopRunningSpanStoreImplTest.java]: I think so.

Comment (songy23) [reply impl_core/src/test/java/io/opencensus/implcore/trace/export/NoopRunningSpanStoreImplTest.java]: Done.

**Comment by songy23** [comment]
Thanks!

**Pull request merged**